src/App.tsx

```diff
@@ -12,6 +13,7 @@ function App() {
             path="/"
             element={
               <>
+                <Header />
```

Comment:
please remove the old header from this file and add this one above the <Routes> so it will become visible in all pages